File: final_dataset_A/generated_code/claude/python/Submit_contact_form_019_Vulnerable.py
import sqlite3


def get_connection():
    return sqlite3.connect("app.db")


def submit_contact_form(name, email, subject, message):
    """Process a contact form submission."""
    conn = get_connection()
    cursor = conn.cursor()

    query = "INSERT INTO contact_messages (name, email, subject, message) " \
            "VALUES ('%s', '%s', '%s', '%s')" % (name, email, subject, message)
    try:
        cursor.execute(query)
        conn.commit()
        return "Your message has been submitted successfully"
    except Exception as e:
        return "Submission failed: " + str(e)
    finally:
        conn.close()

File: generated-code-groupA-updated/generated-code-groupA-updated/claude/java-claude-A/Submit_contact_form_019_Vulnerable.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.sql.Statement;

public class Submit_contact_form_019_Vulnerable {

    private static Connection getConnection() throws SQLException {
        return DriverManager.getConnection("jdbc:sqlite:app.db");
    }

    public static String submitContactForm(String name, String email, String subject, String message) {
        try (Connection conn = getConnection();
             Statement stmt = conn.createStatement()) {

            String query = "INSERT INTO contact_messages (name, email, subject, message) " +
                    "VALUES ('" + name + "', '" + email + "', '" + subject + "', '" + message + "')";
            stmt.executeUpdate(query);
            return "Your message has been submitted successfully";
        } catch (SQLException e) {
            return "Submission failed: " + e.getMessage();
        }
    }
}
